apps/sequencer/src/feeds/votes_result_sender.rs

```diff
@@ -193,7 +195,21 @@ async fn try_send_aggregation_consensus_trigger_to_reporters(
             } else {
                 info!("Network `{net}` is enabled; initiating second round consensus");
             }
-
+            if provider_settings
+                .get_contract_config(GNOSIS_SAFE_CONTRACT_NAME)
+                .is_none()
+            {
+                info!("Network `{net}` not configured for second round consensus - skipping");
+                continue;
+            }
```

Comment:
I see that we have above (Line 182) another check `if provider_settings.safe_address.is_none()` that we might want to reconsider and potentially remove.